WebDriverAgentLib/Commands/FBOrientationCommands.m

```diff
@@ -65,7 +66,7 @@ + (NSArray *)routes
 + (id<FBResponsePayload>)handleGetRotation:(FBRouteRequest *)request
 {
     XCUIDevice *device = [XCUIDevice sharedDevice];
-    UIDeviceOrientation orientation = device.orientation;
+    UIInterfaceOrientation orientation = request.session.activeApplication.interfaceOrientation;
```

Comment:
this is also a breaking change, but in my experience `device.orientation` always resulted in a 0-degree rotation, meaning it was useless.